test/parallel/test-tls-getcipher.js

```diff
@@ -1,37 +1,37 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
 if (!common.hasCrypto) {
   common.skip('missing crypto');
   return;
 }
-var tls = require('tls');
+const tls = require('tls');
 
-var fs = require('fs');
-var cipher_list = ['AES128-SHA256', 'AES256-SHA256'];
-var cipher_version_pattern = /TLS|SSL/;
-var options = {
+const fs = require('fs');
+const cipher_list = ['AES128-SHA256', 'AES256-SHA256'];
+const cipher_version_pattern = /TLS|SSL/;
+const options = {
   key: fs.readFileSync(common.fixturesDir + '/keys/agent2-key.pem'),
   cert: fs.readFileSync(common.fixturesDir + '/keys/agent2-cert.pem'),
   ciphers: cipher_list.join(':'),
   honorCipherOrder: true
 };
 
-var server = tls.createServer(options,
+const server = tls.createServer(options,
                               common.mustCall(function(cleartextStream) {}));
```

Comment:
@Ethan-Arrowood The argument (`common.mustCall...`)on line 22 should line up with the argument on line 21 (`options`). So line 22 needs to be indented two more spaces. I'm surprised this isn't caught by the linter.